jetson/percep/pcl.hpp

```diff
@@ -7,19 +7,29 @@
 /* --- Compare Line Class --- */
 //Functor that indicates where a point is in
 //relation to a line in 2D space
+//recieves angle off y axis
 class compareLine {
 public:
+    int xIntercept;
```

Comment:
I love this name change from `b`. Can you do the same thing for `m` -> `slope` (or `angle` as you allude to below)?